TGC/decisions/SPA.txt

```diff
@@ -1,5 +1,60 @@
-political_decisions = {
-#from SPA.txt
+﻿political_decisions = {
```

Comment:
The whole file is in the incorrect encoding, it needs to be re-saved as win 1252.